### DISCUSSION TITLE
Vivo New (backgroundBrandSecondary)

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1339] [Proposals]
### Goal

Alterar a cor do token backgroundBrandSecondary.

### Context

Os tokens backgroundBrand e backgroundBrandSecondary estão recebendo a mesma cor e gostaríamos de fazer essa alteração para aplicação nas interfaces.

De: #660099 / Para: #751AA3 (vivoPurpleLight90)

### Impact

Adicionar essa nova cor em um token existente nos oferece mais opções para explorar contrastes diferentes.

### Discussion Questions

Como podemos fazer essa alteração? Seria responsabilidade do time da Vivo?

**Comment by yceballost** [comment]
Hi Anne!

For this type of change you don't need to open a discussion, you are free to change colors as you wish, you can have this ownership. 

 There are two ways to include this change in Mística:
 1. [Open an issue](https://github.com/Telefonica/mistica-design/issues/new?assignees=&labels=&projects=&template=new_issue.md&title=) directly and Mística team will work on it
 2. Including this change by yourself! We very much believe that teams should have the freedom to change more superficial changes that do not have a major impact, such as colours. You can create a pull request to this repository including this change.

Feel free to choose your favorite option

@aweell, maybe is interesting to include a guide to help designers to contribute to this repo as a designer. @annefranca if you think this guide could be useful we could do it!

**Reply by annefranca** [reply]
Olá @yceballost acredito que esse guia vai ser muito útil, pois eu não sabia que esse tipo de alteração poderia ser feito por aqui.

**Reply by yceballost** [reply]
Great, we will include asap, by this way, you will have more autonomy to change little things

**Reply by annefranca** [reply]
De qualquer forma, abri uma issue.

**Reply by yceballost** [reply]
perfect!
